internal/interceptor/server.go

```diff
@@ -20,6 +20,11 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const (
```

Comment:
@michaelsauter Aren't we coupling the interceptor to a specific version of the ods-start task? I guess for now it's OK, but it would be nice to pass that info as params to avoid include it as part of the interceptor Docker image.

Reply:
This is an intern solution yes. I'm not sure on the way forward yet. I think the version of the interceptor and the start/finish task should be the same. maybe we bake it into the image via build arg / env, and then it's set by default but can be overridden in the pod if need be.